Sources/Auth/Types/AuthRequestParams.swift

```diff
@@ -0,0 +1,33 @@
+
+import Foundation
+import WalletConnectUtils
+
+struct AuthRequestParams: Codable, Equatable {
+    let requester: Requester
+    let payloadParams: AuthPayload
+
+    static var tag: Int {
+        return 3000
+    }
+}
+
+extension AuthRequestParams {
+    struct Requester: Codable, Equatable {
+        let publicKey: String
+        let metadata: AppMetadata
+    }
+}
+
+// TODO - temporarly duplicated - moved do utils in concurrent PR
```

Comment:
I believe that, if this is moved to Utils, we'll have to expose it via a `typealias` in each SDK. Remember to test it if you do